build-configuration/pom.xml

```diff
@@ -180,7 +180,7 @@
       <version.org.wildfly.core>5.0.0.Final</version.org.wildfly.core>
       <version.osgi>4.3.1</version.osgi>
       <version.pax.exam>4.11.0</version.pax.exam>
-      <version.protostream>4.2.1.Final</version.protostream>
+      <version.protostream>4.2.2-SNAPSHOT</version.protostream>
```

Comment:
Final, not Snapshot please

Reply:
Done. but CI does not love it yet. Need to re-run it.